packages/fluentui/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -1296,4 +1297,61 @@ describe('Dropdown', () => {
       expect(renderSelectedItem).toBeCalledTimes(value.length);
     });
   });
+
+  describe('disabled', () => {
+    it('allows no action on the trigger button', () => {
+      const { clickOnTriggerButton, focusTriggerButton, getItemNodes, triggerButtonNode, keyDownOnTriggerButton } = renderDropdown({
+        disabled: true
+      });
+
+      expect(triggerButtonNode).toHaveAttribute('disabled');
+
+      clickOnTriggerButton();
+
+      expect(getItemNodes()).toHaveLength(0);
+
+      focusTriggerButton();
+
+      expect(triggerButtonNode).not.toHaveFocus();
+
+      keyDownOnTriggerButton('ArrowDown');
+
+      expect(getItemNodes()).toHaveLength(0);
+    });
+
+    it('allows no action on the search input', () => {
+      const { keyDownOnSearchInput, clickOnSearchInput, focusSearchInput, getItemNodes, searchInputNode } = renderDropdown({
+        disabled: true,
+        search: true
+      });
+
+      expect(searchInputNode).toHaveAttribute('disabled');
+
+      keyDownOnSearchInput('ArrowDown');
+
+      expect(getItemNodes()).toHaveLength(0);
+
+      focusSearchInput();
+
+      expect(searchInputNode).not.toHaveFocus();
+
+      clickOnSearchInput();
+
+      expect(searchInputNode).not.toHaveFocus();
+    });
+
+    it('allows no action on the toggle indicator', () => {
+      const { clickOnToggleIndicator, toggleIndicatorNode, getItemNodes } = renderDropdown({
+        disabled: true
+      });
+
+      clickOnToggleIndicator();
+
+      expect(getItemNodes()).toHaveLength(0);
+
+      toggleIndicatorNode.focus();
+
```

Comment:
A newline after each action looks a bit strange, is it intent?

Reply:
Not trying to make it strange intentionally, just separating test steps from expects.